### ISSUE TITLE
[css-typed-om] Link "CSS Values"

### ISSUE THREAD

**Issue by annevk** [w3c/css-houdini-drafts#693]
In https://drafts.css-houdini.org/css-typed-om/#stylevalue-objects it'd be good to link the definition of CSS Values.

**Comment by tabatkins** [comment]
That wasn't intended to be a reference to any particular term; the title-casing appears to have been a mistake. I've just lowercased "values".

**Comment by annevk** [comment]
I think even so you need to link the term since it's not otherwise defined.

**Comment by tabatkins** [comment]
I don't believe we have a definition of the term "CSS values", and I'm not sure how or why anyone even remotely familiar with the web platform would need such a term?  They're values from CSS, the definition is the literal English reading.

**Comment by annevk** [comment]
If these objects present a reification of some underlying model, you'd expect the underlying model to have defined terms as well that make up the model.

I don't think familiarity should be required for reading a specification. Everything needs to be grounded in primitives.
